test/units/testsuite-82.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Please do not create new TEST-82, but add the test as a sub test of TEST-19. That is, please rename testsuite-82.sh as testsuite-19.cleanup-slice.sh or so.

Note, TEST-82 is also will be used #27435.